don't install Intel MIC runtine components (mpi-mic-rt), cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/3793
author: Kenneth Hoste (HPC-UGent)
--- parallel_studio_xe_2019_update1_composer_edition_for_fortran/pset/mediaconfig.xml.orig	2018-11-16 12:34:28.375110983 +0100
+++ parallel_studio_xe_2019_update1_composer_edition_for_fortran/pset/mediaconfig.xml	2018-11-16 12:43:36.082555264 +0100
@@ -1047,7 +1047,7 @@
     </Layers>
     <InstallDir>${COMPLIB_ROOT}</InstallDir>
     <GUID>1540897359989</GUID>
-    <Component depend="33.3;1.1;1.3;1.91;1.11;1.12;1.14;1.16;1.7;1.27;1.24;133.2" id="1" invisible="0" mandatory="0" platform="INTEL64">
+    <Component depend="33.3;1.1;1.3;1.91;1.11;1.12;1.14;1.16;1.7;1.27;1.24" id="1" invisible="0" mandatory="0" platform="INTEL64">
       <Name>Intel Fortran Compiler for Intel(R) 64</Name>
       <Description>Intel Fortran Compiler 19.0 Update 1</Description>
       <Name locale="ja_JP.UTF-8">インテル(R) Fortran コンパイラー (インテル(R) 64)</Name>
